src/components/Signup/Signup.tsx

```diff
@@ -97,10 +97,10 @@ const Signup: React.FC<SignupProps> = ({ isShow, returnShow }) => {
     <>
       <Modal show={show} onHide={handleClose}>
         <Modal.Header closeButton>
-          <Modal.Title>Sign Up</Modal.Title>
+          <Modal.Title>Log In</Modal.Title>
```

Comment:
Make it `Login or Sign Up`

Reply:
Okay

Reply:
completed [833f61e](https://github.com/Tanay-ErrorCode/lupo-skill/pull/146/commits/833f61e1f5396f23b4aa4fd4f8078698688b5d13)